README.md

```diff
@@ -2,6 +2,38 @@
 
 This is a list of sql commands to add,insert and filter fictional animal elements inside a DB by the name 'vet_clinic'.
 
+# INDEXING
+
+**<span style="color:red;">animals_id</span>**
+
+**BEFORE**
+
+![animals_id](./Before/before_performance_animalsid.png)
+
+**AFTER**
+
+![animals_id](./After/after_performance_animalsid.png)
+
+**<span style="color:red;"><storng>vets_id</strong></span>**
+
+**BEFORE**
+
+![vets_id](./Before/before_performance_vetsid.png)
+
+**AFTER**
+
+![vets_id](./After/after_performance_vetsid.png)
+
```

Comment:
- [ ] For this query `EXPLAIN ANALYZE SELECT * FROM visits WHERE vets_id=2;` has the execution time in the before which is less than the after, whereas it should be the other way, (means the after should be optimized in a way that it's less than the before, would you address this issue?

Reply:
First of all, thank you very much for accepting the project as is. The answer is I don't know but the sequential scan seems to work better in this case unlike the previous two who were really boosted. According to my research, Seq scan on postgres runs Parnell which is fast already, another reason can be caching or the fact that i'm using a high end processor that seq scan is fast already on it and that heap indexed scan doesn't reach it. I'm not sure. I tried to drop index from the previous run (animals_id) to no avail still. do you have any tips? Would be much appreciated

Reply:
@descholar-ceo  we also tried the indexing with order and it was still the same time